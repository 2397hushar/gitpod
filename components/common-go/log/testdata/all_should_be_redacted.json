{
	"instanceId": "21473566-c15c-4ecd-9356-d1e8fdff2797",
	"envvars": [
		"this should not be here"
	],
	"ssh_public_keys": [
		"this should not be here"
	],
	"env": [
		"this should not be here"
	],
	"auth_password": "this should not be here",
	"email": "this should not be here",
	"owner_token": "this should not be here"
}