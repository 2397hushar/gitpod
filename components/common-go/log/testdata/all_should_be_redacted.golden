{
    "level": "panic",
    "msg": "",
    "req": {
        "auth_password": "[REDACTED]",
        "email": "[REDACTED]",
        "env": [],
        "envvars": [],
        "instanceId": "21473566-c15c-4ecd-9356-d1e8fdff2797",
        "owner_token": "[REDACTED]",
        "ssh_public_keys": []
    },
    "time": "0001-01-01T00:00:00Z"
}